### PR TITLE
Added score-k8s to documentation

### PR DESCRIPTION
**Pull request by sujaya-sys**
Thank you for submitting this pull request! We appreciate you spending the time to work on these changes.

## What is the motivation?

Adding score-k8s reference to developer docs after making https://github.com/score-spec/score-k8s public 

## Have you read the [Contributing Guidelines](CONTRIBUTING.md)?

- [x] I have read the [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: content/en/docs/score implementation/included/install-score-k8s-bash.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: "score-k8s install on macOS"
+description: "Installation for score-k8s on macOS."
+headless: true
+toc_hide: true
+---
+
+**1.** Download the latest compressed Tar file for your operating system from the [GitHub release page](https://github.com/score-spec/score-k8s/releases). By default, the tarball will be saved to your `~/Downloads` directory. If you choose to use a different location, you'll need to change that in the following steps.
```
Comment (astromechza) [inline content/en/docs/score implementation/included/install-score-k8s-bash.md:8]: Not sure about this line. The tarball will be saved wherever you end up downloading it right?

File: content/en/docs/score implementation/included/install-score-k8s-bash.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+```bash
+score-k8s_x.y.z_osx-amd64.tar.gz
```
Comment (astromechza) [inline content/en/docs/score implementation/included/install-score-k8s-bash.md:11]: Are we missing a `curl` or `wget` call here?

File: content/en/docs/score implementation/included/install-score-k8s-curl-mac.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+title: "score-k8s install on macOS"
+description: "Installation for score-k8s on macOS."
+headless: true
+toc_hide: true
+---
+
+**1.** Download the latest release from the [GitHub release page](https://github.com/score-spec/score-k8s/releases):
+
+```bash
+wget https://github.com/score-spec/score-k8s/releases/download/<x.y.z>/score-k8s_<x.y.z>_<os_system>.tar.gz
```
Comment (astromechza) [inline content/en/docs/score implementation/included/install-score-k8s-curl-mac.md:11]: did you mean curl here instead?

File: content/en/docs/score implementation/score-k8s.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+The help command provides information on all commands.
+
+```bash
+  score-k8s help [command] [flags]
+```
+
+#### Flags
+
+The `help` command can be combined with the following flags:
```
Comment (astromechza) [inline content/en/docs/score implementation/score-k8s.md:164]: This is a bit weird... `help --help`? maybe we should exclude this in the docs

**Review by astromechza — CHANGES_REQUESTED** [review]
Looks fine to me. one thing to fix.

**Review by TobiasBabin — CHANGES_REQUESTED** [review]
Looking good, some minor comments re: the commands only.

**Comment by sujaya-sys** [comment]
Ready for re-review!

File: content/en/docs/score implementation/score-k8s.md (hunk 4 above)
Comment (sujaya-sys) [reply content/en/docs/score implementation/score-k8s.md]: Removed 👍

**Review by astromechza — APPROVED** [review]

**Comment by sujaya-sys** [comment]
@TobiasBabin looks like i need your re-review as well before being able to merge it. Thanks a lot :)

**Review by TobiasBabin — APPROVED** [review]
🎉

**Pull request merged**